Bottom: ed41a92560f35ddff4df8f0496f6023b77aa14da
Top:    ae0de6e8b6c6c80010feaf805c4af11cf36f8552
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-25 10:06:42 +0800

Refresh of autofs-5.0.7-fix-compilation-of-lookup_ldap_c-without-sasl.patch

---

diff --git a/modules/lookup_ldap.c b/modules/lookup_ldap.c
index 904cc21..7648a56 100644
--- a/modules/lookup_ldap.c
+++ b/modules/lookup_ldap.c
@@ -66,7 +66,7 @@ struct ldap_search_params {
 
 static int decode_percent_hack(const char *, char **);
 
-#ifdef HAVE_SASL
+#ifdef WITH_SASL
 static int set_env(unsigned logopt, const char *name, const char *val)
 {
 	int ret = setenv(name, val, 1);
@@ -587,7 +587,7 @@ static LDAP *do_connect(unsigned logopt, const char *uri, struct lookup_context
 {
 	LDAP *ldap;
 
-#ifdef HAVE_SASL
+#ifdef WITH_SASL
 	if (ctxt->extern_cert && ctxt->extern_key) {
 		set_env(logopt, ENV_LDAPTLS_CERT, ctxt->extern_cert);
 		set_env(logopt, ENV_LDAPTLS_KEY, ctxt->extern_key);
@@ -1398,7 +1398,7 @@ static void free_context(struct lookup_context *ctxt)
 		defaults_free_searchdns(ctxt->sdns);
 	if (ctxt->dclist)
 		free_dclist(ctxt->dclist);
-#ifdef HAVE_SASL
+#ifdef WITH_SASL
 	if (ctxt->extern_cert)
 		free(ctxt->extern_cert);
 	if (ctxt->extern_key)
